src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -42,8 +42,8 @@ public function __construct(\Twig_Environment $environment, TemplateNameParserIn
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Inject the escaping
-     *             strategy on Twig_Environment instead
+     * @deprecated since version 2.3, to be removed in 3.0.
+     *             Inject the escaping strategy on \Twig_Environment instead.
      */
     public function setDefaultEscapingStrategy($strategy)
```

Comment:
please add a deprecation warning in it (TwigBundle does not call this method anymore. It sets the strategy directly in Twig)